notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -72,7 +73,7 @@
    "outputs": [],
    "source": [
     "## Change this cell to change the kernel function throughout\n",
-    "\n",
+    "kernel_params = {\"kernel\": \"rbf\", \"gamma\": 1.0}\n",
     "kernel_func = gaussian_kernel\n",
     "kernel_type = 'gaussian'"
```

Comment:
Are these used anywhere? If not, delete.

Reply:
Yes, they are. 
![image](https://user-images.githubusercontent.com/56018055/99814694-9d80c100-2b49-11eb-9b7d-95324528681c.png)
![image](https://user-images.githubusercontent.com/56018055/99814702-a1144800-2b49-11eb-939d-e3f7253befb7.png)
![image](https://user-images.githubusercontent.com/56018055/99814756-b7ba9f00-2b49-11eb-850f-bf168acb1cd1.png)